src/anonymousUserTracking/anonymousUserMerge.ts

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable class-methods-use-this */
+import { ENDPOINT_MERGE_USER } from '../constants';
+import { baseIterableRequest } from '../request';
+import { IterableResponse } from '../types';
+
+export type MergeApiParams = {
+  sourceEmail: string | null;
+  sourceUserId: string | null;
+  destinationEmail: string | null;
+  destinationUserId: string | null;
+};
+
+export class AnonymousUserMerge {
+  mergeUser(
+    sourceUserId: string | null,
+    sourceEmail: string | null,
+    destinationUserId: string | null,
+    destinationEmail: string | null
+  ): Promise<void> {
+    const mergeApiParams: MergeApiParams = {
+      sourceUserId,
+      sourceEmail,
+      destinationUserId,
+      destinationEmail
+    };
+    return this.callMergeApi(mergeApiParams);
+  }
+
+  private callMergeApi(data: MergeApiParams): Promise<void> {
```

Comment:
Hi

We are getting `Merging known user profiles is forbidden with this api key type` error for this. We are using `Web (Client-side - JWT required)` api key and I assume others are doing the same. 

What is the recommended approach here?

I don't think using server side for client side sdk is safe
and yes, merging users indeed should be server side feature. Otherwise, any user can merge any 2 user profiles.

I guess a safe solution would be to create another api that only allows merging anonymous user ids into a user profile (for which the JWT was generated). Not sure how it would work without JWT.